redfish_utilities/update.py

```diff
@@ -141,6 +157,20 @@ def multipart_push_update( context, image_path, targets = None ):
     """
 
     # Get the update service
+    if timeout is None:
+        timeout = 5
+        file_size = 0
+        try:
+            if os.path.isfile(image_path) is True:
+                file_size = get_size(image_path, "mb")
+            else:
+                raise FileNotFoundError
+        except FileNotFoundError:
+            raise RedfishUpdateFileNotFoundError( "{} is not found".format(image_path) )
```

Comment:
Do we need our own exception for this? The built-in FileNotFoundError exception should be well understood.